test/morpho-blue/TestLeverageDeleverageSnippets.sol

```diff
@@ -14,7 +14,7 @@ contract LeverageDeleverageTest is BaseTest {
     using MarketParamsLib for MarketParams;
     using SharesMathLib for uint256;
 
-    address internal USER;
+    address internal BORROWER_;
```

Comment:
why the _ is needed?

Reply:
BORROWER is already defined in base tests, and there is previous approval, so I prefer this rather than BORROWER_2, do you think we should do BORROWER_2?

Reply:
We don't even need this address since this contract inherits from `BaseTest` which already declare a `BORROWER` address.